master/changes.html

```diff
@@ -340,7 +340,10 @@ <h3 id="structure">Document Structure chapter</h3>
 
   <li>Removed the currentView and useCurrentView properties on <a>SVGSVGElement</a>.</li>
 
-  <li>Added <a>SVGUnknownElement</a> for handling unknown elements in the svg namespace.</li>
+  <!-- <li>Added <a>SVGUnknownElement</a> for handling unknown elements in the svg namespace.</li> -->
```

Comment:
```suggestion
  <li>Added SVGUnknownElement for handling unknown elements in the svg namespace (later removed).</li>
```

Keep the history in the changes section.

Reply:
Similar comments for the rest of the edits to this file.

Reply:
That is fine with me. Other specs more clearly differ between the individual TRs/CRs which makes it easier to follow the changes from a historical perspective.

Reply:
I made the similar edits in this section